src/components/CurrentDirectors/CurrentDirectors.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div id="current-directors">
+
+    <v-card flat>
+      <!-- Header -->
+      <div class="header-container">
+        <v-icon color="app-dk-blue">mdi-account-multiple-plus</v-icon>
+        <label class="font-weight-bold pl-2">Current Directors</label>
+      </div>
+
+      <!-- Instructional Text -->
+      <div class="instructional-text pt-10 pb-15 px-4">
+        To change directors, please use the Change feature in the Current Directors list on you business dashboard.
+      </div>
+
+      <div class="list-container px-4">
+        <!-- List Display Section -->
+        <div id="people-roles-list" v-if="currentDirectors.length > 0">
+          <!-- List Headers -->
+          <v-row class="people-roles-list-header list-item__subtitle pb-3" no-gutters>
+            <v-col class="pr-5" v-for="(title, index) in tableHeaders" :key="index">
+              <span class="directors-title">{{ title }}</span>
+            </v-col>
+          </v-row>
+
+          <!-- List Content -->
+          <v-row
+            class="people-roles-content py-3"
+            v-for="(orgPerson, index) in currentDirectors"
+            :key="index"
+            no-gutters
+          >
+            <!-- Name + Badge -->
+            <v-col class="text-truncate pr-5">
```

Comment:
How wide are each of these v-cols?

How does this look when the screen is narrowed? We don't support mobile very well atm but we do try to make our columns a bit responsive.

Reply:
I based this off the table for People and Roles so it's setup the exact same way I believe

Reply:
OK I looked at the tables used in People and Roles and I don't see special v-col spacing. Please fix it.

As for the col width, you're right, it's not specified on the other page either. I dunno. If it matches the ux design then you're good.